CHANGELOG.md

```diff
@@ -12,7 +12,8 @@
 - The `--overwrite` flag instructs [`operator-sdk bundle create`](./doc/cli/operator-sdk_bundle_create.md) to overwrite metadata, manifests, and `bundle.Dockerfile`. ([#2715](https://github.com/operator-framework/operator-sdk/pull/2715))
 - [`operator-sdk bundle validate`](./doc/cli/operator-sdk_bundle_validate.md) now accepts either an image tag or a directory arg. If the arg is a directory, its children must contain a `manifests/` and a `metadata/` directory. ([#2737](https://github.com/operator-framework/operator-sdk/pull/2737))
 - Add support to release SDK arm64 binaries and images. ([#2742](https://github.com/operator-framework/operator-sdk/pull/2715))
- 
+- Add annotation `helm.sdk.operator-framework.io/upgrade-force: "true"` to allow force resources replacement (`helm upgrade --force`) for Helm based-operators. (`helm upgrade --force`) ([#2773](https://github.com/operator-framework/operator-sdk/pull/2773))
```

Comment:
```suggestion
- Add annotation `helm.operator-sdk/upgrade-force: "True"` to allow force resources replacement (`helm upgrade --force`) for Helm based-operators. (`helm upgrade --force`) ([#2773](https://github.com/operator-framework/operator-sdk/pull/2773))
```

Reply:
I just notice that the annotation is not following the project standard. So, could you please replace it for we are able to merge this one?

So, could you please replace :

> helm.sdk.operator-framework.io/upgrade-force: "true"

with:

> helm.operator-sdk/upgrade-force: "True"
 
Really thank you for your terrific contribution.